tests/admin_views/tests.py

```diff
@@ -1544,6 +1544,17 @@ def test_add_view(self):
         self.assertEqual(mail.outbox[0].subject, 'Greetings from a created object')
         self.client.get(reverse('admin:logout'))
 
+        # Check that the addition was logged correctly
```

Comment:
It would be nice if the test verified the change message for a form with formsets.